<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>checkDialog</class>
 <widget class="QDialog" name="checkDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>701</width>
    <height>468</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>确认导出多个文件</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QListWidget" name="listLW">
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="allCheckedBTN">
       <property name="minimumSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="myrc.qrc">
         <normaloff>:/icon/icons/green_ok_24px_1075415_easyicon.net.png</normaloff>:/icon/icons/green_ok_24px_1075415_easyicon.net.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clearCheckedBTN">
       <property name="minimumSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="myrc.qrc">
         <normaloff>:/icon/icons/red_delete_24px_1075470_easyicon.net.png</normaloff>:/icon/icons/red_delete_24px_1075470_easyicon.net.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="insertCheckedBTN">
       <property name="minimumSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="myrc.qrc">
         <normaloff>:/icon/icons/selin_32.png</normaloff>:/icon/icons/selin_32.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="sortBTN">
       <property name="minimumSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="myrc.qrc">
         <normaloff>:/icon/icons/shot_32px.png</normaloff>:/icon/icons/shot_32px.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="applyBTN">
       <property name="text">
        <string>确  认</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelBTN">
       <property name="text">
        <string>取  消</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="myrc.qrc"/>
 </resources>
 <connections/>
</ui>
